packages/client/base/src/services/api/helpers/validateApiKeyAndGetCrossmintBaseUrl.ts

```diff
@@ -0,0 +1,17 @@
+import { APIKeyEnvironmentPrefix, validateAPIKey } from "@crossmint/common-sdk-base";
+
+import { CROSSMINT_DEV_URL, CROSSMINT_PROD_URL, CROSSMINT_STG_URL } from "../../../consts";
+
+const urlMap: Record<APIKeyEnvironmentPrefix, string> = {
+    development: CROSSMINT_DEV_URL,
+    staging: CROSSMINT_STG_URL,
+    production: CROSSMINT_PROD_URL,
+};
+
+export function validateApiKeyAndGetCrossmintBaseUrl(apiKey: string) {
```

Comment:
Should this go in the base package instead, where the other API key validation related code lives. Server SDKs probably want this too right?

Reply:
done